### PR TITLE
[FGA-98] Add `schema convert` and `schema apply` to FGA module

### PR DESCRIPTION
**Pull request by atainter**
Usage:
Convert a schema to json
```bash
workos fga schema convert ./schema.txt --to json 
```
Write to a file
```bash
./bin/workos-cli fga schema convert ./schema.txt --to json --output raw > test.json
```

Apply a schema
```bash
workos-cli fga schema apply ./schema.txt
```
Apply a schema with verbose output and fail on warnings
```bash
workos-cli fga schema apply ./schema.txt -v --strict
```

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -115,6 +115,8 @@ github.com/subosito/gotenv v1.6.0 h1:9NlTDc1FTs4qu0DDq7AEtTPNw6SVm7uBMsUCUjABIf8
 github.com/subosito/gotenv v1.6.0/go.mod h1:Dk4QP5c2W3ibzajGcXpNraDfq2IrhjMIvMSWPKKo0FU=
 github.com/workos/workos-go/v4 v4.16.0 h1:7BPvVKHI8qhOY/lH4EbBY2CtQs1CDJxXDbfTnYNma3g=
 github.com/workos/workos-go/v4 v4.16.0/go.mod h1:CwpXdAWhIE3SxV49qBVeYqWV8ojv0A0L9nM1xnho4/c=
+github.com/workos/workos-go/v4 v4.21.0 h1:pEoAJzCsBPU46dL6/PwwwS5BrBV8LWOZQp0mERrRPCc=
```
Comment (kkajla12) [inline go.sum:118]: Can you run a `go mod tidy`? It looks like we have an old version of the Go SDK still listed in this file.

File: internal/cmd/fga.go [hunk 2]
```diff
@@ -65,6 +66,15 @@ func init() {
 	queryCmd.Flags().String("order", "", "order in which a list of results should be returned (asc or desc)")
 	fgaCmd.AddCommand(queryCmd)
 
+	// schema
+	convertSchemaCMD.Flags().String("to", "json", "output to (schema or json)")
+	convertSchemaCMD.Flags().String("output-format", "pretty", "output format (pretty or raw). use raw for machine-readable output or writing to a file")
+	schemaCmd.AddCommand(convertSchemaCMD)
+	applySchemaCmd.Flags().BoolP("verbose", "v", false, "print extra details about the request")
+	applySchemaCmd.Flags().Bool("fail-on-warnings", false, "fail if there are warnings")
```
Comment (kkajla12) [inline internal/cmd/fga.go:74]: Thoughts on naming this flag `--strict`?

File: internal/cmd/fga.go [hunk 3]
```diff
@@ -65,6 +66,15 @@ func init() {
 	queryCmd.Flags().String("order", "", "order in which a list of results should be returned (asc or desc)")
 	fgaCmd.AddCommand(queryCmd)
 
+	// schema
+	convertSchemaCMD.Flags().String("to", "json", "output to (schema or json)")
+	convertSchemaCMD.Flags().String("output-format", "pretty", "output format (pretty or raw). use raw for machine-readable output or writing to a file")
```
Comment (kkajla12) [inline internal/cmd/fga.go:71]: Thoughts on making this a boolean flag called `--raw`?

Comment (atainter) [reply internal/cmd/fga.go:71]: That's what I originally named it 😃 Yes, I can change it back. I originally thought it was a bit ambiguous.

File: internal/cmd/fga.go (hunk 2 above)
Comment (atainter) [reply internal/cmd/fga.go:74]: that's fine with me

File: internal/cmd/fga.go (hunk 3 above)
Comment (kkajla12) [reply internal/cmd/fga.go:71]: Alternatively, we could shorten it to just `--output`:
```shell
--output raw
--output pretty
```

**Review by kkajla12 — APPROVED** [review]

**Pull request merged**